gap/semigroups/semirms.gi

```diff
@@ -11,6 +12,28 @@
 # this file contains methods for every operation/attribute/property that is
 # specific to Rees 0-matrix semigroups.
 
+InstallImmediateMethod(IsFinite,
+IsReesZeroMatrixSubsemigroup and HasGeneratorsOfSemigroup, 0,
+function(R)
+  local P;
+  P := ReesMatrixSemigroupOfFamily(FamilyObj(R.1));
+  if HasIsFinite(P) and IsFinite(P) then
+    return true;
+  fi;
+  TryNextMethod();
+end);
+
+InstallImmediateMethod(IsFinite,
+IsReesMatrixSubsemigroup and HasGeneratorsOfSemigroup, 0,
+function(R)
+  local P;
+  P := ReesMatrixSemigroupOfFamily(FamilyObj(R.1));
+  if HasIsFinite(P) and IsFinite(P) then
+    return true;
+  fi;
+  TryNextMethod();
+end);
+
 #############################################################################
```

Comment:
I think these immediate methods for `IsFinite` are superseded by changes I made in the GAP library. Can you double check please?